### PR TITLE
fix: funnel group tests

### PR DESCRIPTION
**Pull request by EDsCODE**
## Problem

- ee test_funnel group tests weren't running because the check was missing after #12137 

## Changes

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

**Comment by EDsCODE** [comment]
@timgl I think you had something in mind here with how the function was inherited based on the comment?

File: ee/clickhouse/queries/funnels/test/breakdown_cases.py [hunk 1]
```diff
@@ -32,8 +33,31 @@ def _create_groups(self):
             create_group(team_id=self.team.pk, group_type_index=1, group_key="org:5", properties={"industry": "random"})
 
         def _assert_funnel_breakdown_result_is_correct(self, result, steps: List[FunnelStepResult]):
-            # set in TestFunnelBreakdown
-            raise NotImplementedError()
+            def funnel_result(step: FunnelStepResult, order: int) -> Dict[str, Any]:
```
Comment (EDsCODE) [inline ee/clickhouse/queries/funnels/test/breakdown_cases.py:36]: don't dup

**Review by mariusandra — APPROVED** [review]
whoops

**Comment by timgl** [comment]
yeah it was all a little bit awkward, but this is fine.

**Pull request merged**